operator/hack/common.sh

```diff
@@ -34,26 +35,27 @@ function apply_operator_manifests() {
   catalog_source_crd="$("${ROOT_DIR}/operator/hack/retry-kubectl.sh" < /dev/null get customresourcedefinitions.apiextensions.k8s.io catalogsources.operators.coreos.com -o yaml)"
   local disable_security_context_config="# "
   local has_scc_key
+  local operator_channel
+  local install_plan_approval
+  local starting_csv
   has_scc_key="$(yq eval '[.. | select(key == "grpcPodConfig")].[].properties | has("securityContextConfig")' - <<< "$catalog_source_crd")"
   if [[ "$has_scc_key" == "true" ]]; then
       disable_security_context_config=""
   fi
 
   if [[ "${USE_MIDSTREAM_IMAGES}" == "true" ]]; then
-    # Get Operator channel from json for midstream
-    operator_channel=$(< midstream/iib.json jq -r '.operator.channel')
-  env -i PATH="${PATH}" \
-    INDEX_VERSION="${index_version}" OPERATOR_VERSION="${operator_version}" NAMESPACE="${operator_ns}" OPERATOR_CHANNEL="${operator_channel}" \
-    IMAGE_TAG_BASE="${image_tag_base}" DISABLE_SECURITY_CONTEXT_CONFIG="${disable_security_context_config}" \
-    envsubst < "${ROOT_DIR}/operator/hack/operator-midstream.envsubst.yaml" \
-    | "${ROOT_DIR}/operator/hack/retry-kubectl.sh" -n "${operator_ns}" apply -f -
+    install_plan_approval="Automatic"
+    starting_csv="null"
   else
+    install_plan_approval="Manual"
+    starting_csv="rhacs-operator.${csv_version}"
+  fi
   env -i PATH="${PATH}" \
-    INDEX_VERSION="${index_version}" OPERATOR_VERSION="${operator_version}" NAMESPACE="${operator_ns}" \
-    IMAGE_TAG_BASE="${image_tag_base}" DISABLE_SECURITY_CONTEXT_CONFIG="${disable_security_context_config}" \
+    INDEX_IMAGE_TAG="${index_image_tag}" STARTING_CSV="${starting_csv}" NAMESPACE="${operator_ns}" OPERATOR_CHANNEL="${operator_channel}" \
+    INDEX_IMAGE_REPO="${index_image_repo}" DISABLE_SECURITY_CONTEXT_CONFIG="${disable_security_context_config}" \
+    INSTALL_PLAN_APPROVAL="${install_plan_approval}" \
```

Comment:
May I ask you to break variables one per line? This way, it will be easier to review changes in this command in the future.
```suggestion
    INDEX_IMAGE_TAG="${index_image_tag}" \
    STARTING_CSV="${starting_csv}" \
    NAMESPACE="${operator_ns}" \
    OPERATOR_CHANNEL="${operator_channel}" \
    INDEX_IMAGE_REPO="${index_image_repo}" \
    DISABLE_SECURITY_CONTEXT_CONFIG="${disable_security_context_config}" \
    INSTALL_PLAN_APPROVAL="${install_plan_approval}" \
```